packages/cashier/src/stores/payment-agent-transfer-store.ts

```diff
@@ -1,10 +1,35 @@
 import { action, computed, observable } from 'mobx';
+import { PaymentAgentDetailsResponse, PaymentAgentListResponse, PaymentAgentTransferRequest } from '@deriv/api-types';
 import { routes } from '@deriv/shared';
 import Constants from 'Constants/constants';
+import { TRootStore, TWebSocket } from 'Types';
 import ErrorStore from './error-store';
 
+type TConfirm = {
+    amount?: number;
+    client_id?: string;
+    client_name?: string;
+    description?: string;
+};
+
+type TReceipt = {
+    amount_transferred?: number;
+    client_id?: string;
+    client_name?: string;
+};
+
+type TMinMaxWithdrawal = {
+    min_withdrawal?: number;
+    max_withdrawal?: number;
+};
+
+type TMinMax = {
+    min?: number;
+    max?: number;
+};
+
 export default class PaymentAgentTransferStore {
-    constructor({ WS, root_store }) {
+    constructor(public WS: TWebSocket, public root_store: TRootStore) {
         this.root_store = root_store;
```

Comment:
As we assign WS and root_store with the public key, I think we can remove this line and the line after this.

Reply:
@hamid-deriv Yes but then we will need to ignore eslint error 😅 That’s why I keep them for now as later we will be refactoring these stores hopefully.

Do you want me to remove them? 🙇🏻

Reply:
It's OK :+1: